### PR TITLE
fixed broken img tags on readme & added one more contributors

### PR DESCRIPTION
**Pull request by whb07**
fixed broken links for img tags for core contributors. Added two more to the 'core contributors' list : Anna & Tatu in reference to the [wiki.status.im](https://github.com/status-im/wiki.status.im/)

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #...

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
...

### Steps to test:
- Open Status
- ...
- Step 3, etc.

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: <ready|wip>

### CODE REVIEW

**Comment by flexsurfer** [comment]
hi @whb07 why do you think that links are broken? works fine for me, and we should use images from the site not from wiki, and Tatu not in the team anymore, we should remove it from wiki, thanks

**Comment by whb07** [comment]
screen shot 2017-06-05 at 7 14 39 am
As you can see...the image links on the README were broken. To that extent, I did remove Tatu from the updated README, I added Anna, and added .png files for the group under /images/contacts next to where jarrad's .png was.

File: README.md [hunk 1]
```diff
@@ -58,36 +58,39 @@ Currently the fastest way to get your hands on a binary is to join our Early Acc
 
 ## Core Contributors
 
-<img src="https://github.com/status-im/status-site/raw/master/src/img/new-site/jarrad.png" width="80px"></img>
+<img src="https://github.com/status-im/status-react/blob/develop/images/contacts/jarrad.png" width="80px"></img>
```
Comment (whb07) [inline README.md]: uploaded all the other images to be in a collection next to jarrad.png

File: README.md [hunk 2]
```diff
@@ -58,36 +58,39 @@ Currently the fastest way to get your hands on a binary is to join our Early Acc
...
 
-<img src="https://github.com/status-im/status-site/raw/master/src/img/new-site/victor.png" width="80px"></img>
+<img src="https://github.com/status-im/status-react/blob/develop/images/contacts/victor.png" width="80px"></img>
 Victor ([@farazdagi](https://github.com/farazdagi)) - Golang one-man-army.
 
-<img src="https://github.com/status-im/status-site/raw/master/src/img/new-site/andrei.png" width="80px"></img>
+<img src="https://github.com/status-im/status-react/blob/develop/images/contacts/andrei.png" width="80px"></img>
 Andrei ([@andmironov](https://github.com/andmironov)) - Visual design and user experience conjurer.
 
+<img src="https://github.com/status-im/status-react/blob/develop/images/contacts/anna.png" width="80px"></img>
```
Comment (whb07) [inline README.md]: added anna to the group

**Comment by flexsurfer** [comment]
looks like this PR is outdated, fixed in this PR https://github.com/status-im/status-react/pull/1364

**Pull request closed without merging**